scripts/devnet-algo/rotate_dbg.sh

```diff
@@ -0,0 +1,10 @@
+cd /dbg
+for i in $(ls | grep [[:digit:]]-  | awk -F- '{print $1}' | uniq); do 
+  in_use=$(lsof | grep $i-[[:digit:]] ; echo $?)
```

Comment:
I am scared of the race condition when the daemon has just closed `6-20` and is about to open `6-21`. We know it will never modify anything but the last one, so I think it is simpler to just only care about that

Reply:
On this one I'll get a return like this from: ls | grep [[:digit:]]-  | awk -F- '{print $1}' | uniq:
15
16
17
18
It will basically iterate along the existing sections, it won't look file by file.
So if there's any 15- file in use, it skips it and moves to the next section so all 15-x.json files won't be gziped until 15 workflow is actually finished.